mui-org/material-ui
# [Legacy] Removes 'TouchTap' in favor of `Click` throughout library and docs

### This resolves #9052.

We had _most_ of our library covered with regards to using replacing `onTouchTap` with `onClick`, However, we neglected to get the names changed that resemble `onActionTouchTap`.

- Also cleans up several internal component functions that had a funny signature like this:

```
  onClick={onHandleTouchTap}
```

This also updates the docs in all spots necessary
- Props
- Variable names
- PropTypes descriptions

## To Test

Run this locally and double-check that site is usable in the same way as [the real site](http://www.material-ui.com/#/) is usable. 🤓 

**N.B. This will introduce breaking changes.** 😆
